### ISSUE TITLE
[BUG] "Companion Unable to Start" on Windows 10

### ISSUE THREAD

**Issue by CymaSpace** [bitfocus/companion#2474]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Running with 3.0.0 RC1
![image](https://github.com/bitfocus/companion/assets/7018464/48174297-0c31-480b-97a1-e914eb8a1ae2)
"Companion is unable to start" on Windows 10. We had removed an older version of v2 prior to this and rebooted. 

### Steps To Reproduce

1. Run companion
2. Error message and quits on start.

### Expected Behavior

Companion should run as normal without error and quit.

### Environment (please complete the following information)

```markdown
- OS: Windows 10.
- Browser: Chrome (latst)
- Companion Version: 3.0.0 RC1
```

**Comment by Julusian** [comment]
please provide the log folder `%APPDATA%/companion/logs`

**Comment by CymaSpace** [comment]
Sure thing! BTW 2.4.2 works fine (just tried reinstalling that). 
[companion-2023-06-09.0.log](https://github.com/bitfocus/companion/files/11709352/companion-2023-06-09.0.log)

**Comment by Julusian** [comment]
This has been a deep dive, with a fresh windows install that needs to be thrown away due to my messing with it.  

I have a suspicion about the problem. 
Is your machine part of a corporate domain?
In the start menu, are you able to run "Windows PowerShell"?

I was so close to giving up, out of ideas, but finally found that there is a `spawn` being done outside of our code, in a library without the appropriate error handler.  
Reproducing this was a nightmare, I had to start deleting system files so that `powershell.exe` was not found. According to google, it isnt possible to uninstall this, but it is possible to disable it via GPO in a domain. It maybe is possible to trigger this by enabler some stricter policy in powershell too, I was unable to reproduce it via this route.

I managed to manually edit our compiled code to add this missing error handler, then companion was able to start!
I have made a fix for this, could you try beta 3.0.0-5950 or later?
